improver_tests/acceptance/test_wxcode.py

```diff
@@ -177,3 +178,23 @@ def test_no_lightning(tmp_path):
     ]
     run_cli(args)
     acc.compare(output_path, kgo_path)
+
+
+@pytest.mark.parametrize(
+    "wxtree,expected",
+    (
+        ("wx_decision_tree_1h.json", "Decision tree OK\nRequired inputs are:"),
+        ("wx_decision_tree_3h.json", "Decision tree OK\nRequired inputs are:"),
+        ("bad_wx_decision_tree.json", "Unreachable node 'unreachable'"),
+    ),
+)
+def test_trees(wxtree, expected):
```

Comment:
I think this should be a unit test not an acceptance tests. The rest of the utility you've modified has been unit tested not acceptance tested.

Reply:
The point of this test is to check that the decision tree used in the other acceptance tests is valid, as well as making sure that the option works when called from the CLI.
I've just updated this branch with the latest changes from master, as Ben's change to unify the decision trees would have broken this after merging.

Reply:
Ok, that makes sense for the valid decision tree, but I think the failure should still be tested in the same place as all the other failures this bit of code can detect, which is in the unit tests.